components/automate-ui/src/app/helpers/auth/authorized.ts

```diff
@@ -47,8 +49,8 @@ export class AuthorizedChecker {
 
   public setPermissions(allOf: CheckObj[], anyOf: CheckObj[]): void {
     // store these for handling the async responses
-    this.allOf = allOf;
-    this.anyOf = anyOf;
+    this.allOf = this.inflate(allOf);
+    this.anyOf = this.inflate(anyOf);
```

Comment:
These operations will be transparent for non-parameterized endpoints, but will now inflate the parameterized ones as needed below.